tanner/config.py

```diff
@@ -11,7 +11,8 @@
                    'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
                    'EMULATORS': {'root_dir': '/opt/tanner'},
                    'SQLI': {'db_name': 'tanner.db'},
-                   'LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'DEBUG_LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'ERROR_LOGGER': {'log_file': '/opt/tanner/tanner.err'},
```

Comment:
like in your PR. I didn't want to copy so I left it my way. But I'll fix it now ;)